core/src/main/java/de/danoeh/antennapod/core/util/playback/Playable.java

```diff
@@ -177,11 +174,18 @@
 
     void setChapters(List<Chapter> chapters);
 
+    /**
+     * Returns the location of the image or null if no image is available.
+     * The location can either be a URL or a local path.
+     */
+    String getImageLocation();
```

Comment:
Please clarify the difference between image url and image location in javadoc. Url=item url from the feed. Location=item url from the feed or feed image if there is no item url or embedded cover

Reply:
Ok, I wrote a more detailed description now. Please check if it is correct.